webapp/sass/responsive/_mobile.scss

```diff
@@ -209,11 +209,12 @@
 
         .post__header {
             margin-bottom: 0;
+            padding-right: 50px;
```

Comment:
Ah, yes, I see what you're saying. Digging into it a little bit, it may be worth considering the following options. What are your thoughts?
We keep everything on the same line, resulting in this look on iphone 6 as a minimum screensize without breaking:
![screen shot 2016-11-08 at 12 16 01 pm](https://cloud.githubusercontent.com/assets/3498300/20115477/3cc61caa-a5ad-11e6-9caf-b832a43f426a.png)

Or we can allow breaking as part of the design for post headers with lots of content:
![screen shot 2016-11-08 at 12 15 38 pm](https://cloud.githubusercontent.com/assets/3498300/20115505/564e8ff4-a5ad-11e6-9b51-57b349111d8f.png)